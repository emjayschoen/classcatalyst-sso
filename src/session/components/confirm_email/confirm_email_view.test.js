import React from 'react';
import { shallow } from 'enzyme/build';
import ConfirmEmailView from './confirm_email_view';

describe('<ConfirmEmailView />', () => {
  let subject;

  beforeEach(() => {
    subject = shallow(<ConfirmEmailView confirmEmail={jest.fn()} />);
  });

  it('renders correctly', () => {
    expect(subject).toMatchSnapshot();
  });
});
