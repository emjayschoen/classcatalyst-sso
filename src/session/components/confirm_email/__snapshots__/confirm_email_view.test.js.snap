// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ConfirmEmailView /> renders correctly 1`] = `
<styled.div>
  <p>
    Foo
  </p>
</styled.div>
`;
